.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: puzzle-lib CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm run build --if-present
```

Comment:
?
Why `run build --if--present` maybe compile?

Reply:
The compile step is run automatically after `npm install` and also before `npm test` so it shouldn't be required explicitly.

This was part of the default template, I've deleted it and updated to use `npm ci` which is the recommended command for CI runs.